engine/access/rpc/engine.go

```diff
@@ -369,12 +396,14 @@ func (e *Engine) serveREST() {
 	e.addrLock.Unlock()
 
 	e.log.Debug().Str("rest_api_address", e.restAPIAddress.String()).Msg("listening on port")
+	ready()
 
 	err = e.restServer.Serve(l) // blocking call
 	if err != nil {
 		if errors.Is(err, http.ErrServerClosed) {
 			return
 		}
-		e.log.Error().Err(err).Msg("fatal error in REST server")
+		e.log.Err(err).Msg("fatal error in REST server")
```

Comment:
isnt the pattern here: `log.Error().Err(err).Msg("")`? https://github.com/rs/zerolog#error-logging

Reply:
`logger.Err(...)` sets the level to error automatically when a non-nil error is passed -- it's a shorthand for what you've suggested: https://pkg.go.dev/github.com/rs/zerolog#Logger.Err

(In contrast to `event.Err(...)`, which just adds the error field to log event.)

Reply:
Did not know, TY.